uint8 starter_id
uint32 qid

uint8[] required

# required.size() == 0  -->  Empty question
uint8[] question

Answer[] answers
